lambda/input/src/main/scala/InputLambdaMain.scala

```diff
@@ -118,7 +118,7 @@ class InputLambdaMain extends RequestHandler[S3Event, Unit] with DocId with Zone
   def getObjectVersion(rec:S3EventNotification.S3EventNotificationRecord) = rec.getS3.getObject.getVersionId match {
```

Comment:
Well worth having a quick unit-test for this - just check the three cases, null, empty string and valid concent

Reply:
Done.